app/assets/templates/pages/myRecipes.html

```diff
@@ -1,51 +1,67 @@
+    </div> -->
     <div class="row">
-      <div class="filter-tags col-xs-10">
-        <div class="filter-tags-heading">Searched tags:</div>
-        <div class="filter-tag">Chicken <i class="fa fa-times"></i></div>
-        <div class="filter-tag">Peas <i class="fa fa-times"></i></div>
-      </div>
-      <a class="create-recipe-button col-xs-2 btn btn-warning" href="/#/create">
+      <!-- <button ng-click="resetFinder()" data-status="open" class="reset-button btn btn-default col-xs-6">
+        Reset Filters
+      </button> -->
+      <!-- <button ng-click="submitParams()" data-status="open" class="submit-button btn btn-warning col-xs-3">
+        Submit
+      </button> -->
+      <button ng-click="toggleFinder()" data-status="closed" class="hide-finder-button btn btn-default col-xs-3">
+        Show Finder
+      </button>
+      <div class="col-xs-6 myrecipes-title">My Recipes</div>
+      <a class="create-recipe-button btn btn-default col-xs-3" href="/#/create">
         Add a Recipe
       </a>
     </div>
 
+
     <div class="row">
       <!-- the whole filter section -->
-      <div class="filter col-xs-12 col-sm-7">
-
+      <div class="filter col-xs-12 col-sm-6 hide">
         <!-- keyword search -->
-        <div class="filter-keyword-wrapper row">
-          <input type="text" class="filter-keyword-search col-xs-offset-1 col-xs-10" placeholder="Search Ingredients">
+        <div class="row">
+          <button ng-click="resetFinder()" data-status="open" class="reset-button btn btn-default col-xs-12">
+            Reset Filters
+          </button>
         </div>
+        <form ng-submit="addKeyword()" class="filter-keyword-wrapper row">
+          <div class="filter-tags col-xs-12">
+            <div class="filter-tags-heading">Search for:</div>
+            <div class="keyword-collection">
+              <!-- list of keywords -->
+              <div ng-click="removeKeyword($index)" ng-repeat='keyword in search_parameters.keywords track by $index' class='filter-tag'>
+                {{ keyword }}
+                <!-- <i class='fa fa-times'></i> -->
+                <button type="button" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              </div>
+            </div>
+          </div>
+          <!-- keyword input -->
+          <input type="text" class="input-sm filter-keyword-search col-xs-8" placeholder="Search by..." ng-model="newKeyword">
+          <select class="col-xs-2 input-sm">
+            <option>Ingredient</option>
+            <option>Keyword</option>
+          </select>
+          <button type="submit" class="add-keyword-button btn btn-default col-xs-2">Add</button>
+        </form>
 
         <!-- filter by meals -->
         <div class="filter-meals-wrapper row">
-          <div class="filter-meals-heading col-xs-12 col-md-3">Meal:
+          <div class="filter-meals-heading col-xs-12"> <i class="fa fa-cutlery"></i> Meals 
```

Comment:
it would be cleaner to put `<i class="fa fa-cutlery"></i> Meals ` in a separate line